metaflow/client/core.py

```diff
@@ -2004,17 +1926,106 @@ def runs(self, *tags):
 
         Parameters
         ----------
-        tags : string
+        tags : str
             Tags to match.
 
         Returns
         -------
-        Iterator[Run]
+        Iterable[Run]
             Iterator over `Run` objects in this flow.
         """
         return self._filtered_children(*tags)
 
 
+class Metaflow(object):
```

Comment:
Any reasons to move this object to the end? Are there any diffs here that I should be looking into?

Reply:
it needed `Flow` iirc.

Reply:
There should be no change.